packages/expo-print/src/Print.types.ts

```diff
@@ -25,10 +25,10 @@ export type PrintOptions = {
    */
   printerUrl?: string;
   /**
-   * **Available on iOS only.** Alternative to `html` option that uses [UIMarkupTextPrintFormatter](https://developer.apple.com/documentation/uikit/uimarkuptextprintformatter)
-   * instead of WebView. Might be removed in the future releases.
+   * **Available on iOS only.** Alternative to default option that uses [UIMarkupTextPrintFormatter](https://developer.apple.com/documentation/uikit/uimarkuptextprintformatter)
+   * instead of WebView, but it doesnt't display images. Might be removed in the future releases.
```

Comment:
```suggestion
   * **Available on iOS only.** Alternative to `html` option that uses [UIMarkupTextPrintFormatter](https://developer.apple.com/documentation/uikit/uimarkuptextprintformatter)
   * instead of WebView, but it doesn't display images. Might be removed in the future releases.
```